tensorflow/tools/ci_build/linux/rocm/run_cpu.sh

```diff
@@ -41,7 +41,7 @@ if [ -f /usertools/cpu.bazelrc ]; then
           --bazelrc=/usertools/cpu.bazelrc \
           test \
           --config=sigbuild_local_cache \
-          --config=pycpp \
+          --config=nonpip \
```

Comment:
are we using nonpip instead of pycpp?

Reply:
yes, pretty sure that what upstream does for CPU - there are less tests though